fractions/microprofile/microprofile-restclient/README.adoc

```diff
@@ -0,0 +1,40 @@
+= MicroProfile Rest Client
+
+== CDI Interceptors Support
+
+In general, Rest Client proxies are not created by the CDI container and therefore method invocations do not pass through CDI interceptors.
+{Thorntail}, however, implements a non-portable feature which allows to associate business method interceptors (i.e. interceptors denoted by `@AroundInvoke`) with a Rest Client proxy using interceptor bindings.
```

Comment:
My understanding is that our docs don't like parens. I'd suggest either rephrasing, or replacing the term "business method interceptors" with just "`@AroundInvoke` interceptors". It's more straightforward and everyone refers to them using the annotation anyway.

Reply:
> My understanding is that our docs don't like parens

What? I've never heard this could be a problem... 

Well, "business method interceptors" is an official term and I'd like to use it...

Reply:
> We indeed try to avoid using parentheses as much as possible...

And why? Is there a technical reason or is it a good practice in docs team?

> Feel free to object if I am changing the meaning to something that is not true...

Yes, your proposal does not sound right from CDI/interceptors POV. What about this:

_In Thorntail, however, you can associate business method interceptors (denoted by `@AroundInvoke` annotation) with a Rest Client proxy. You can do so by using interceptor bindings. This feature is non-portable._

Reply:
> And why? Is there a technical reason or is it a good practice in docs team?

Text in parentheses breaks the mental flow of the sentence they are in, and therefore make the text harder to read. We try to distract the reader as little as possible. Bear in mind that the readers of the docs are often not very good at English, so every bit of mental capacity not dedicated to solving the actual problem at hand is wasted.